src/components/AppsCard/AppsCard.css

```diff
@@ -191,3 +191,8 @@
   outline: 2px solid #ff0000;
   color: #ff0000;
 }
+
+.AppDeleteModel .ModalChildWrap {
+  min-width: 21rem;
```

Comment:
@neelxie i'm not for fixed layouts

Reply:
i expected u to make the button wrapper a flex container with a `justify-content` of `space-between`

Reply:
@mrwagaba that is still the underlying of this container.